src/groupInfo/components/GroupGalleryUI.vue

```diff
@@ -213,8 +221,19 @@ export default {
     hasOtherGroupsToShow () {
       return this.expanded && this.filteredOtherGroups.length > 0
     },
+    paddingTopLeft () {
+      // when the sidebar is open on a wide enough desktop screen
+      // we set the padding so the markers aren't displayed underneath the sidebar
+      if (this.expanded && window.innerWidth > 767 && this.$q.platform.is.desktop) {
```

Comment:
Where does `767` come from? Maybe it could be a named constant or something similar?

Reply:
That one I just left as it was before. I wondered where it came from before... I can have a look. And yes, at least a constant with some hint what it means.

Reply:
I couldn't understand where it came from, it doesn't correspond to quasar breakpoints, also isn't a reactive value, and so I removed it entirely... seems to still work ok :)